partner_base_multicompany_default/views/partner_view.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2023 Moduon Team S.L.
+     Copyright 2024 Camptocamp SA
+     License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0) -->
+<data>
+
+    <record id="view_partner_form" model="ir.ui.view">
```

Comment:
Shouldn't this stay in the `partner_account_*` module?
If you keep it here, we have a missing dependency because this requires `account`.

Reply:
Good catch, this was already moved and not present in the manifest. This is a left over file.